src/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/Helpers.cs

```diff
@@ -118,6 +118,32 @@ public static byte[] FixupKeyParity(this byte[] key)
             }
             return oddParityKey;
         }
+
+        public static void ConvertIntToByteArray(uint dwInput, ref byte[] counter)
```

Comment:
Declaring it public was my bad, I'll make it internal. Will change the parameter values. Thanks for the review guys :)